src/main/java/th/in/gamer/user/response/UserStatusResponse.java

```diff
@@ -0,0 +1,35 @@
+package th.in.gamer.user.response;
```

Comment:
What is the propose of this bean ?
btw you should omit `Serializable` implementation.